tdrs-frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -33,25 +31,14 @@ function PrivateRoute({ children, title, history, path }) {
     if (authenticated) {
       dispatch(clearAlert())
     }
-  }, [authenticated, authLoading, dispatch, history])
+  }, [authenticated, authLoading, dispatch, navigate])
 
   return authenticated ? (
-    <Route path={path}>
-      <PrivateTemplate title={title}>
-        {children}
-        <IdleTimer />
-      </PrivateTemplate>
-    </Route>
+    <PrivateTemplate title={title}>
```

Comment:
Note we are no longer wrapping this with `Route`, and it is handled higher up in the tree